metagraph/src/cli/config/config.cpp

```diff
@@ -949,21 +944,21 @@ void Config::print_usage(const std::string &prog_name, IdentityType identity) {
             fprintf(stderr, "Available options for alignment:\n");
             fprintf(stderr, "\t-o --outfile-base [STR]\t\t\t\tbasename of output file []\n");
             fprintf(stderr, "\t   --json \t\t\t\t\toutput alignment in JSON format [off]\n");
-            fprintf(stderr, "\t   --align-both-strands \t\t\treturn best alignments for either input sequence or its reverse complement [off]\n");
+            fprintf(stderr, "\t   --align-one-strand \t\t\t\tdo not align backwards from a seed on basic-mode graphs [off]\n");
             fprintf(stderr, "\t   --align-alternative-alignments \t\tthe number of alternative paths to report per seed [1]\n");
             fprintf(stderr, "\t   --align-min-path-score [INT]\t\t\tthe minimum score that a reported path can have [0]\n");
-            fprintf(stderr, "\t   --align-edit-distance \t\t\tuse unit costs for scoring matrix [off]\n");
             fprintf(stderr, "\t   --align-max-nodes-per-seq-char [FLOAT]\tmaximum number of nodes to consider per sequence character [12.0]\n");
             fprintf(stderr, "\t   --align-max-ram [FLOAT]\t\t\tmaximum amount of RAM used per alignment in MB [200.0]\n");
+            fprintf(stderr, "\t   --align-xdrop [INT]\t\t\t\tthe maximum difference between the current and the best alignment [27]\n");
+            fprintf(stderr, "\t   --align-fraction-of-top [FLOAT]\t\tduring extension, use this fraction of the current best alignment as a lower bound on the alignment score [0.8]\n");
```

Comment:
```suggestion
            fprintf(stderr, "\t   --align-fraction-of-top [FLOAT]\t\tmin score relative to the current best alignment for keeping paths during extension [0.8]\n");
```

Reply:
"fraction" is confusing, we use it to refer to subset sizes in other cases. Maybe `--min-relative-score` ?

Reply:
or `--rel-score-cutoff`

Reply:
for the help message, how about
```min score relative to the current best alignment to use as a lower bound for subsequent extensions```

Reply:
either is fine